sdk/storage/azure-storage-files-datalake/src/datalake_utilities.cpp

```diff
@@ -98,6 +98,10 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake { nam
     blobOptions.ApiVersion = options.ApiVersion;
     blobOptions.CustomerProvidedKey = options.CustomerProvidedKey;
     blobOptions.EnableTenantDiscovery = options.EnableTenantDiscovery;
+    if (options.Audience.HasValue())
+    {
+      blobOptions.Audience = Blobs::Models::BlobAudience(options.Audience.Value().ToString());
```

Comment:
Does dfs audience work for blob? Will this break interoperability?

Reply:
Yes, I've confirmed with Jason Fan and Xiaoning Liu. it's interoperable